e2e/favorites.spec.ts

```diff
@@ -0,0 +1,13 @@
+import { expect, test } from '@playwright/test'
+import { testAuth } from './fixtures/authenticatedUserPages'
+import { goToDashboard } from './helpers/testUser'
+
+test.describe('Favorites', () => {
+  testAuth('should be visible', async ({ userPage }) => {
```

Comment:
Use the `userPage` test fixture, rather than configuring the 'loggedin' project to have a `page` with user's `storageState`. Allows tests to be organized by subject, rather than by authentication state, i.e. file renamed from `favorites.loggedin.spec.ts` to `favorites.spec.ts`.